BraveRewardsUI/Ads/AdsViewController.swift

```diff
@@ -332,3 +332,13 @@ extension AdsViewController {
     })
   }
 }
+
+extension AdsViewController {
+  public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    if #available(iOS 13.0, *) {
```

Comment:
I forget, do you need to call super on this? Also didn't realize Swift let you override functions in extensions now :P

Reply:
Yeah you must call super.